src/AppBundle/PullRequests/Listener.php

```diff
@@ -58,7 +71,7 @@ public function checkForTableDescription(PullRequest $pullRequest)
         $bodyParser = new BodyParser($pullRequest->getBody());
 
         $validationErrors = $this->validator->validate($bodyParser);
-        if (count($validationErrors) > 0) {
+        if (\count($validationErrors) > 0) {
```

Comment:
Move to use `use function count`

Reply:
php-cs-fixer won't accept it I already tried ;) Do you think I should change the rule though?

Reply:
Ping @PrestaShop/prestashop-core-developers

Reply:
What is the reason of adding the backslash here? :thinking:

Reply:
To make php-cs-fixer happy, that's one of its rules